src/frontend/src/lib/config/send.config.ts

```diff
@@ -0,0 +1,21 @@
+import { WizardStepsSend } from '$lib/enums/wizard-steps';
+import type { WizardSteps } from '@dfinity/gix-components';
+
+export const sendWizardSteps = (i18n: I18n): WizardSteps => [
+	{
+		name: WizardStepsSend.TOKENS_LIST,
+		title: i18n.send.text.send
+	},
+	{
+		name: WizardStepsSend.SEND,
```

Comment:
I guess it just duplicates temporarly the similar existing steps right?

Reply:
correct! until we merge into one (another PR) or we use one of them to create the others

Reply:
Great. Can you add a TODO?

Reply:
Done:

- PR #1944